docs/spec_draft.md

```diff
@@ -1280,6 +1281,58 @@ For boolean element type, the behavior is same as [stablehlo.or](#stablehloor).
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.map
+
+### Semantics
+
+Performs a map function `computation` over the `inputs` along the `dimensions`
+and produces a `result` tensor where each element is the result of `computation`
+applied to the corresponding elements in `inputs`.
+
+More formally, `result[i0, ..., iR-1] = computation(inputs0[i0, ..., iR-1], `
+`..., inputsN-1[i0, ..., iR-1])`.
```

Comment:
What does "par1" stand for in "For example: Map(op1, op2, op3, computation, par1) maps elem_out <- computation(elem1, elem2, elem3, par1) at each (multi-dimensional) index in the input arrays to produce the output array" from https://www.tensorflow.org/xla/operation_semantics#map?

Reply:
"par1" seems to be part of a deprecated argument currently not used. The up to date semantics of HLO is in sync with that of StableHLO.

Reply:
It should say:
"For example: Map(op1, op2, op3, computation) maps elem_out <- computation(elem1, elem2, elem3) at each (multi-dimensional) index in the input arrays to produce the output array"